typescript/test/unit/multi_proto.ts

```diff
@@ -17,16 +17,11 @@ import { execSync } from 'child_process';
 import * as fs from 'fs';
 import * as path from 'path';
 import * as rimraf from 'rimraf';
+import { compareToBaseline } from '../util';
 
 const cwd = process.cwd();
 
 const OUTPUT_DIR = path.join(cwd, '.multi-proto-test-out');
```

Comment:
Maybe make it `.test-out-keymanager` and `.test-out-showcase` to make it easier to find?

Reply:
Sure!